### PR TITLE
refactor: PHP 7.3 compatibility

### LINKED ISSUE #19: Incompatible with PHP 7.3
Please test it thoroughly before publishing it to production.

https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.mixed

The feature is introduced in PHP 8.0, yet it's used widely in the code.

Example error: ( ! ) Fatal error: Uncaught TypeError: Return value of KiteConnect\KiteConnect::getPositions() must be an instance of KiteConnect\mixed, instance of stdClass returned in D:\www\trading\vendor\zerodha\phpkiteconnect\src\KiteConnect.php on line 648

There might be more issues with PHP 7.3. Please test it thoroughly with PHP 7.3.

### PR DESCRIPTION
**Pull request by pushpak1300**
For PHP 7.3 backward compatibility which resolves #19 .
Tested this on PHP7.3 locally and all tests are passing.

Screenshot from 2021-04-21 12-43-00

@vividvilla @ranjanrak

### CODE REVIEW

**Comment by ranjanrak** [comment]
Thanks for the PR. I Will go through this over weekend and update here.

**Comment by pushpak1300** [comment]
> Thanks for the PR. I Will go through this over weekend and update here.

Any updates?

**Comment by ranjanrak** [comment]
@pushpak1300 
We should also remove the **Mixed_** Docblock type enforced in [invalidateRefreshToken](https://github.com/zerodha/phpkiteconnect/blob/master/src/KiteConnect.php#L388) and [modifyOrder](https://github.com/zerodha/phpkiteconnect/blob/master/src/KiteConnect.php#L493) functions, when mixed type is already added in docblock comment above [these functions](https://github.com/zerodha/phpkiteconnect/blob/master/src/KiteConnect.php#L379).

**Comment by pushpak1300** [comment]
@ranjanrak I have removed it

**Comment by pushpak1300** [comment]
@ranjanrak any updates?

**Comment by ranjanrak** [comment]
LGTM. @vividvilla, you can merge this.

File: src/KiteConnect.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 use DateTimeZone;
 use Exception;
 use GuzzleHttp\Client;
```
Comment (krish-damani) [inline src/KiteConnect.php:11]: Do we still need this?

Comment (pushpak1300) [reply src/KiteConnect.php:11]: Yes, Missed it. will change it.
Thanks ! 👍🏼

**Pull request closed without merging**